docs/guide/welcome/community.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 0
+---
+
+# Community
+
+Prefect welcomes issues, contributions and discussion from all users regardless of background or experience level.  [cicdw](https://github.com/cicdw) is the main point of contact for the Prefect repository.
+
+## Discussion
+
+- for feature requests or bug reports, please open a [new GitHub issue](https://github.com/PrefectHQ/prefect/issues/new)
+- for realtime discussion or help, please visit our [Gitter chat](https://gitter.im/prefectio/prefect)
+- important architechtural and design decisions are memorialized in [Prefect Improvement Notices (PINs)](../PINs/PIN-1-Introduce-PINs.html)
+
+
+## Contributing
+
+Prefect encourages users to contribute in any way they can!  
+
+There are a number of ways you can get started contributing:
+- **issues**: the [issue board](https://github.com/PrefectHQ/prefect) contains issues with the tag ["good first issue"](https://github.com/PrefectHQ/prefect/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) which should be approachable for first time contributors
```

Comment:
I expected this section titled `**issues**` to be about "opening issues to highlight problems". I'd call this one either `**closing issues**` or `**contributing code**` and have a separate header called "issues" for literally issues.

Reply:
Updated; I agree we need a "How to open issues" section, but wasn't sure whether it was worthwhile to tackle right now